libs/barista-components/core/src/option/option.ts

```diff
@@ -62,6 +62,7 @@ export class DtOptionSelectionChange<T> {
     '[class.dt-option-disabled]': 'disabled',
     '(click)': '_handleClick($event)',
     '(keydown)': '_handleKeydown($event)',
+    '(mousemove)': '_handleMouseMove()',
```

Comment:
Attaching a `mousemove` this way will trigger a CD on every move. Please use `zone.runOutsideAngular` and attach the event handler in that callback (maybe via `fromEvent`). Note, When the correct event has been captured and you emit the hovered output you need to wrap the emitting again in a `zone.run`